src/cleaning/clean.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    clean(mesh)
+
+Remove the unused vertices of `mesh`.
+"""
+function clean(mesh)
```

Comment:
Please use a more descriptive name for the function. If it removes unused vertices, then we should probably call it `rmvertices`. More generally, I think we should discuss a "clean" api that implements our current Transform api. It would be nice to be able to create cleaning pipelines for meshes.

Reply:
I will add the unused faces removal to this function, following **VCG**. What about `collect_garbage(mesh)` for the name? Or what?